### PR TITLE
ZL | Rudementary Session Management System for Frontend

### PR DESCRIPTION
**Pull request by ZacharyLangley**
#### Description
This PR:
* Adds a local YAML pointed to port 8000 for Proxy.
* Adds Rudimentary  Cookie Based Session Management.
* Adds Sign out Functionality with Profile Dropdown.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,9 +10,9 @@
 ## Local Web Development
 
 1. Bring up docker-compose. `docker-compose up -d`
-2. Tear down the ingress, it's only slowing you down. `docker-compose down ingress`
-3. Start the ingress locally through the VSCode debugger.
-4. Run local web development environment. `cd web && npm run`
+2. Tear down the ingress, it's only slowing you down. `docker-compose down ingress` or `docker-compose down ingress`
```
Comment (bobcob7) [inline README.md:13]: Both of these statements here look the same.

File: README.md [hunk 2]
```diff
@@ -10,9 +10,9 @@
...
+4. Start the ingress locally through the VSCode debugger.
```
Comment (bobcob7) [inline README.md:14]: Numbering is off

File: cmd/ingress/serve.go [hunk 3]
```diff
@@ -19,7 +19,6 @@ import (
 	"go.uber.org/zap"
 )
 
-//go:embed public
```
Comment (bobcob7) [inline cmd/ingress/serve.go:22]: We still need this for packaging

File: build/ingress/local.yml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+grpc:
+  address: :8000
+webProxyAddress: ""
```
Comment (bobcob7) [inline build/ingress/local.yml:3]: How are you proxing locally if this is null.

File: build/ingress/local.yml [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+clients:
+  authentication:
+    address: http://authentication
```
Comment (bobcob7) [inline build/ingress/local.yml:6]: These domains are docker network specific. These should be localhost with the correct ports.

File: web/src/app/screens/AuthScreen/Forms/Signup/SignupForm/SignupSuccess.tsx [hunk 6]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+
+import language from 'src/common/language/index';
+import { RoutePath } from 'src/app/types/routes';
+import PrimaryButton from '../../../../../../common/components/Button/PrimaryButton/PrimaryButton';
```
Comment (bobcob7) [inline web/src/app/screens/AuthScreen/Forms/Signup/SignupForm/SignupSuccess.tsx:7]: Ooof! I'm looking forward to aliasing these

File: web/src/domain/requests/sessions.ts [hunk 7]
```diff
@@ -12,26 +18,26 @@ const client = createPromiseClient(
...
+            const token = header.get("session")
+            if (token) sessionToken = token
         }});
     return sessionToken;
 }
 
-// export const signOutRequest = async () => {
-//     return;
-// }
+export function* signOutRequest () {
```
Comment (bobcob7) [inline web/src/domain/requests/sessions.ts:30]: Shouldn't this sign the user out server-side too? Don't you need to call something like `client.deleteSession()`? Or is this a TODO?

File: cmd/ingress/serve.go (hunk 3 above)
Comment (ZacharyLangley) [reply cmd/ingress/serve.go:22]: ah for some reason it was throwing errors for me

File: web/src/domain/requests/sessions.ts (hunk 7 above)
Comment (ZacharyLangley) [reply web/src/domain/requests/sessions.ts:30]: this is a TODO

File: web/src/app/screens/AuthScreen/Forms/Signup/SignupForm/SignupSuccess.tsx (hunk 6 above)
Comment (ZacharyLangley) [reply web/src/app/screens/AuthScreen/Forms/Signup/SignupForm/SignupSuccess.tsx:7]: yep imports need adjustment

File: build/ingress/local.yml (hunk 5 above)
Comment (ZacharyLangley) [reply build/ingress/local.yml:6]: I just copied what you had originally to get this proxy going, seems to work regardless of the change

File: build/ingress/local.yml (hunk 4 above)
Comment (ZacharyLangley) [reply build/ingress/local.yml:3]: I just run the vscode command.

**Review by bobcob7 — APPROVED** [review]

**Pull request merged**